extensions/wikia/AdEngine/js/lookup/prebid/adaptersPricesTracker.js

```diff
@@ -5,7 +5,8 @@ define('ext.wikia.adEngine.lookup.prebid.adaptersPricesTracker', [
 ], function (adaptersRegistry, prebid, log) {
 	'use strict';
 
-	var logGroup = 'ext.wikia.adEngine.lookup.prebid.adaptersPricesTracker';
+	var logGroup = 'ext.wikia.adEngine.lookup.prebid.adaptersPricesTracker',
+		validResponseStatusCode = 1;
```

Comment:
we already have errorCode defined in `ext.wikia.adEngine.lookup.prebid.adaptersPerformanceTracker`. I'm thinking about moving those status codes to `ext.wikia.adEngine.wrappers.prebid` - it's strictly related with prebid implementation and I think it makes sense to define it within prebid wrapper